.github/workflows/pyopenms-wheels.yml

```diff
@@ -325,6 +333,8 @@ jobs:
   test:
     needs: [build-win, build-macos, build-lnx]
     runs-on: ${{ matrix.os }}
+    env:
+      SYSTEM_VERSION_COMPAT: 0 #courtesy of https://github.com/actions/setup-python/issues/469 in lieu of an actual solution.
```

Comment:
maybe also say here that it is a workaround for mac neded for XYZ? above is kind of fine because there it specifies on which os it runs.